geometry_msgs/Vector3       eta
geometry_msgs/Vector3       eta_dot
geometry_msgs/Vector3       eta_dot_dot